src/docs/product/data-management-settings/attachment-scrubbing.mdx

```diff
@@ -0,0 +1,130 @@
+---
+title: "Attachment Scrubbing"
+sidebar_order: 4
+keywords: ["pii", "gdpr", "personally identifiable data", "compliance", "minidump"]
+---
+
+
+Events can include attachments. In some cases, attachments may contain PII. [Minidumps](/platforms/native/guides/minidumps) are a common attachment type for native crash reports, and minidumps may contain PII in various sections of the file.
+
```

Comment:
Suggest rephrasing to:
PII can be included in attachments, which are included in events. [Minidumps](/platforms/native/guides/minidumps), a common attachment type for Native crash reports, may contain PII in various sections of the file."